<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Edit</class>
 <widget class="QDialog" name="Dialog_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Company_Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Company_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Position_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Position:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Position_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Status_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Status_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Weblink_Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Weblink">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weblink:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Edit_OK">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset theme="drive-optical"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
